laguinho/routes/datasets.py

```diff
@@ -23,22 +23,17 @@ def publish():
     result = dataset_metadata.load(request.json, unknown=EXCLUDE)
     if dataset_exists(result):
         return jsonify('Dataset already exists'), 409
-    datasets_metadata.insert_one(result)
-    del result['_id']
+    datasets_metadata.insert_one(result.copy())
     return jsonify(result), 201
 
 
 @datasets.route("/datasets", methods=['GET'], strict_slashes=False)
 def get_datasets():
-    dsets = list(datasets_metadata.find())
-    for ds in dsets:
-        del ds['_id']
-
+    dsets = list(datasets_metadata.find({}, {'_id': 0}))
```

Comment:
I took a quick look at the documentation for this, would `.find(projection={'_id': 0})` work as well? Just so we can avoid passing an empty dictionary whose meaning isn't very clear.

Also, why `0`? I saw that in PyMongo's documentation they use `False` instead. I think it's clearer to use a boolean rather than an integer in this case, if I understood correctly :thinking:

Reply:
Hi there, I've just seen this message.

Sure, python allows you to pass it as positional or keyword argument.

And I agree with you about the False, I'm just too used to do it with zeros, but I've never read that part of Pymongo docs.

I will update the PR with these changes.